politeiawww/politeiawww.go

```diff
@@ -280,6 +280,40 @@ func (p *politeiawww) handleProposalDetails(w http.ResponseWriter, r *http.Reque
 	util.RespondWithJSON(w, http.StatusOK, reply)
 }
 
+// handleBatchProposals handles the incoming proposal batch command. It fetches
+// the complete details for a list of proposals.
+func (p *politeiawww) handleBatchProposals(w http.ResponseWriter, r *http.Request) {
+	log.Tracef("handleBatchProposals")
+	var bp www.BatchProposals
+
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&bp); err != nil {
+		RespondWithError(w, r, 0, "handleBatchProposals: unmarshal",
+			www.UserError{
+				ErrorCode: www.ErrorStatusInvalidInput,
+			})
+		return
+	}
+
+	user, err := p.getSessionUser(w, r)
+	if err != nil {
+		if err != ErrSessionUUIDNotFound {
```

Comment:
```suggestion
		// This is a public route so a session might not exist
		if err != ErrSessionUUIDNotFound {
```

Minor nit.  Can you add this comment in?  It makes it a little easier to read.